lib/tools.js

```diff
@@ -236,6 +237,25 @@ api.clone = function(value) {
 };
 
 /**
+ * Assigns a getter to the value of a config key. When the config key is read, 
+ * the getter will execute and compute the configured value. This is 
+ * particularly useful for config values that depend on other config values; 
+ * it removes the need to reset such a value when its dependencies change.
+ * 
```

Comment:
Add one more separate paragraph:

The value can be computed from a function or from a lodash template that will be evaluated using `bedrock.config` for its local variables.